### PR TITLE
Terse stacktraces: no `::__1`, `std::string` as `String`

### PR DESCRIPTION
**Pull request by myrrc**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Remove `::__1` part from stacktraces. Display `std::basic_string<char, ...` as `String` in stacktraces.

### CODE REVIEW

File: programs/server/config.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
             [1]: https://github.com/pocoproject/poco/blob/poco-1.9.4-release/Foundation/include/Poco/Logger.h#L105-L114
         -->
-        <level>trace</level>
+        <level>warning</level>
```
Comment (alexey-milovidov) [inline programs/server/config.xml:24]: This is wrong.

Comment (myrrc) [reply programs/server/config.xml:24]: Ah, just an error, will rebase shortly (before transitioning PR to ready)

**Comment by alexey-milovidov** [comment]
https://s3.amazonaws.com/clickhouse-test-reports/47171/853918226b2f99cb3b1b2943f59767665310ae89/fuzzer_astfuzzermsan/report.html

**Comment by myrrc** [comment]
Fixed. Other test fails look unrelated

**Comment by myrrc** [comment]
Fixed. All other failing test cases (~hung queries and fuzzer fail) look unrelated

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**